store/src/java/com/zimbra/qa/unittest/TestFolders.java

```diff
@@ -17,31 +17,43 @@
 
 package com.zimbra.qa.unittest;
 
-import junit.framework.TestCase;
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertNotNull;
+import static org.junit.Assert.assertNull;
+
+import org.junit.After;
+import org.junit.AfterClass;
+import org.junit.Before;
+import org.junit.BeforeClass;
+import org.junit.Test;
 
 import com.google.common.base.Strings;
-import com.zimbra.common.util.ZimbraLog;
 import com.zimbra.client.ZFolder;
 import com.zimbra.client.ZMailbox;
+import com.zimbra.common.util.ZimbraLog;
 import com.zimbra.cs.account.Provisioning;
 
-public class TestFolders extends TestCase
-{
-    private static String USER_NAME = "user1";
+public class TestFolders {
+    private static String USER_NAME = TestFolders.class.getSimpleName();
     private static String NAME_PREFIX = "TestFolders";
 
     private String mOriginalEmptyFolderBatchSize;
 
-    @Override
-    protected void setUp()
-    throws Exception {
+
+    @BeforeClass
+    public static void init() throws Exception {
+        TestUtil.deleteAccount(USER_NAME);
```

Comment:
if account does not exist, this will log an error in mailbox.log. It is better to check if account exists first.